LICENSE

```diff
@@ -1,21 +1,51 @@
-MIT License
```

Comment:
Need to rebase this branch against `master` to avoid adding already existing changes. Should only have 1 commit.

Reply:
Not fixed in latest commit.